webpack.config.js

```diff
@@ -1,96 +1,109 @@
-const { CleanWebpackPlugin } = require('clean-webpack-plugin');
+const {CleanWebpackPlugin} = require('clean-webpack-plugin');
 const CopyPlugin = require('copy-webpack-plugin');
+const path = require('path');
 
 const commonConfig = {
   devtool: 'eval-cheap-module-source-map',
   resolve: {
     extensions: ['.js', '.jsx'],
+    alias: {
```

Comment:
Is it possible to add a comment that these need to be added to package.json if we want to use them in tests?

Reply:
yes, good call! added a note about this